framework/Browser.js

```diff
@@ -0,0 +1,109 @@
+import { browser } from '@wdio/globals'
+
+class Browser {
+  async open(url) {
+    await browser.url(url)
+  }
+
+  async getCurrentUrl() {
+    return browser.getUrl()
+  }
+
+  async goBack() {
+    await browser.back()
+  }
+
+  async goForward() {
+    await browser.forward()
+  }
+
+  async refreshPage() {
+    await browser.refresh()
+  }
+
+  async getPageTitle() {
+    return browser.getTitle()
+  }
+
+  async getWindowHandle() {
+    return browser.getWindowHandle()
+  }
+
+  async getWindowHandles() {
+    return browser.getWindowHandles()
+  }
+
+  async closeWindow() {
+    await this.browser.closeWindow()
```

Comment:
this.browser is not defined. it should be just browser

Reply:
this is fixed in the other branch and I merged them. Can I do something else about it?